src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2662,22 +2671,28 @@ namespace HVACUnitarySystem {
                         UnitarySystem(UnitarySysNum).CoolVolumeFlowRate(DesignSpecMSHP(MSHPIndex).NumOfSpeedCooling);
                 }
                 UnitarySystem(UnitarySysNum).IdleVolumeAirRate =
-                    UnitarySystem(UnitarySysNum).CoolVolumeFlowRate(1) * DesignSpecMSHP(MSHPIndex).NoLoadAirFlowRateRatio;
+                    UnitarySystem(UnitarySysNum).CoolVolumeFlowRate(NumSpeeds) * DesignSpecMSHP(MSHPIndex).NoLoadAirFlowRateRatio;
```

Comment:
This (and below) should not be using `UnitarySystem(UnitarySysNum).CoolVolumeFlowRate(NumSpeeds)`. The I/O Ref says the no-load fraction will be multiplied by the "the minimum of the cooling and heating supply air flow rate." so that would be `min(UnitarySystem(UnitarySysNum).MaxHeatAirVolFlow,UnitarySystem(UnitarySysNum).MaxCoolAirVolFlow)`

Reply:
Or as you just suggested, for noload this could all be set earlier using MaxNoCoolHeatAirVolFlow which is already set correctly(?) in the sizing routine.

Reply:
I think this could be simplified but won't do that here. The cooling coils are set first, so here it's just based on cooling in case there is no heating coil. Then the heating coils are set later, and there it's based on the min.